test/Makefile.am

```diff
@@ -236,11 +230,6 @@ BimodalExample_gsl_CPPFLAGS = -I$(srcdir)/t04_bimodal $(AM_CPPFLAGS)
 
 # Files to freedom stamp
 srcstamp =
-srcstamp += $(test_gsl_get_set_row_column_SOURCES)
-srcstamp += $(test_gsl_get_min_max_vec_SOURCES)
-srcstamp += $(test_gsl_multiple_rhs_matrix_solve_SOURCES)
-srcstamp += $(test_gsl_power_method_SOURCES)
-srcstamp += $(test_gsl_inverse_power_method_SOURCES)
```

Comment:
Actually, how about I just remove `srcstamp` altogether? The file list for stamping was replaced by [foo in configure.ac](https://github.com/libqueso/queso/blob/dev/configure.ac#L282) and then that variable is used in the license stamping. And actually, the code that does the stamping exists in more than 1 place. How about I do a separate PR that cleans all of that up?